### DISCUSSION TITLE
LLM Filter Retriever for mutli retrieval QA

### DISCUSSION THREAD

**Discussion by Petercerny** [FlowiseAI/Flowise#1797] [Q&A] [answered]
Hi guys, I cannot connect an LLM Filter Retriever with the multi retrieval QA. Is that not possible now or is that a bug? I tried testing it on the conversational Retrieval QA and on the Retrieval QA chain and it works on both of them, but not on the multi. 
Any help is appreciated :)

**Comment by Jaredude** [comment, marked as answer]
Currently, this is not possible. The Multi Retrieval QA Chain currently only accepts a VectorStoreRetriever and the LLM Filter Retriever only outputs LLMFilterRetriever | BaseRetriever. The CRQAC and RQAC both accept a BaseRetriever, which is why those do connect.
